dependencies/FindFastJet.cmake

```diff
@@ -68,28 +68,35 @@ if(${PKGNAME}_CONFIG)
     if ("CGAL" IN_LIST ${PKGNAME}_LIBS)
       find_library(lib_cgal NAMES "CGAL" PATHS $ENV{CGAL_ROOT}/lib NO_DEFAULT_PATH)
       if (NOT lib_cgal)
-        message(FATAL_ERROR "CGAL not found in $ENV{CGAL_ROOT}/lib")
+        if(FastJet_FIND_REQUIRED)
+          message(FATAL_ERROR "CGAL not found in $ENV{CGAL_ROOT}/lib")
+        endif(FastJet_FIND_REQUIRED)
+      else()
+        get_filename_component(dir_cgal ${lib_cgal} DIRECTORY)
+        list(APPEND ${PKGNAME}_LIB_DIRS ${dir_cgal})
       endif()
-      get_filename_component(dir_cgal ${lib_cgal} DIRECTORY)
-      list(APPEND ${PKGNAME}_LIB_DIRS ${dir_cgal})
     endif()
 
     if ("gmp" IN_LIST ${PKGNAME}_LIBS)
       find_library(lib_gmp NAMES "gmp" PATHS $ENV{GMP_ROOT}/lib NO_DEFAULT_PATH)
       if (NOT lib_gmp)
-        message(FATAL_ERROR "GMP not found in $ENV{GMP_ROOT}/lib")
+        if(FastJet_FIND_REQUIRED)
```

Comment:
This part is only reached if fastjet-config is found, and in this case there is a problem if it reports the use of GMP but we cannot find the library.